bpo-39877: Fix PyEval_RestoreThread() for daemon threads (#18811)
python/cpython: Misc/NEWS.d/next/Core and Builtins/2020-03-06-18-30-00.bpo-39877.bzd1y0.rst

Fix :c:func`PyEval_RestoreThread` random crash at exit with daemon threads.  It

> now access directly ``_PyRuntime`` variable rather than using

Reviewer: ```suggestion
now accesses the ``_PyRuntime`` variable  directly instead of using
```

> It now access directly ``_PyRuntime`` variable rather than using

Reviewer: ```suggestion
It now accesses the ``_PyRuntime`` variable directly instead of using
```

> ``tstate->interp->runtime``, since ``tstate`` can be dangling pointer after

Reviewer: ```suggestion
``tstate->interp->runtime``, since ``tstate`` can be a dangling pointer after
```
